tests/app/questionnaire/test_questionnaire_schema.py

```diff
@@ -288,7 +288,7 @@ def test_get_list_item_id_for_answer_id_without_list_item_id(
 
 
 def test_get_list_item_id_for_answer_id_without_repeat_or_list_collector(
-    question_schema
+    question_schema,
```

Comment:
Why are all these commas being added?

Reply:
These were added when I ran black

Reply:
I think you need to remove the global install of black on your system and reinstall with `pipenv install --dev` as per runners readme. The pr shouldn't be reformatting these as they haven't otherwise been changed.

Reply:
OK ill do that